shoreline/manifest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "90e1b0ed-0907-4973-929c-7e7f1be0c4f4",
+  "app_id": "shoreline-integration",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Category::Automation",
+    "Offering::UI Extension"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "description": "Build automations to repair triggered monitors",
+    "title": "Shoreline Web Service Integration",
```

Comment:
```suggestion
    "title": "Shoreline Remediation",
```

Reply:
@ytnytn1 "Shoreline UI" or something similar also works, but the current title exceeds the tile's character limit.